cypress/integration/Crosscutting/PageOptionsTest.feature

```diff
@@ -0,0 +1,104 @@
+            | /about-nci/organization/nci-at-a-glance                                                     | print,email,facebook,twitter,pinterest |
+            | /espanol/infografia-nci                                                                     | print,email,facebook,twitter,pinterest |
+            | /about-cancer/coping/feelings/relaxation/hedge-maze                                         | print,email,facebook,twitter,pinterest |
+
+    Scenario Outline: Negative: Verify that print and font resizer buttons are not visible at tablet and mobile viewpoints
+        Given screen breakpoint is set to "<breakpoint>"
+        When user is navigating to "<url>"
+        Then page options are displayed
+        And "<pageOption1>" button is not displayed
+        Examples:
+            | url                                                                                         | breakpoint | pageOption1        |
+            | /about-cancer/coping/feelings/relaxation                                                    | tablet     | font resizer,print |
+            | /espanol/hoja-informativa-vitamina-d                                                        | mobile     | font resizer,print |
+            | /about-cancer/coping/feelings/relaxation/loukissas-jennifer                                 | tablet     | font resizer,print |
+            | /news-events/cancer-currents-blog/2019/hpv-vaccine-presidents-cancer-panel-improving-uptake | mobile     | font resizer,print |
+            | /espanol/noticias/temas-y-relatos-blog/2017/tobacco-corrective-es                           | tablet     | font resizer,print |
+            | /types/breast/research                                                                      | mobile     | font resizer,print |
+            | /espanol/tipos/seno/investigacion                                                           | tablet     | font resizer,print |
+            | /about-cancer/coping/feelings/relaxation/dfharvard                                          | mobile     | font resizer,print |
+            | /about-cancer/coping/feelings/relaxation/2019-investigators-site                            | tablet     | font resizer,print |
+            | /news-events/press-releases/2019/annual-report-nation-2018                                  | mobile     | font resizer,print |
+            | /espanol/noticias/comunicados-de-prensa/2019/informe-anual-a-la-nacion-1975-2014            | tablet     | font resizer,print |
+            | /about-cancer/treatment/side-effects/fatigue/fatigue-pdq                                    | mobile     | font resizer,print |
+            | /espanol/cancer/tratamiento/efectos-secundarios/fatiga/fatiga-pdq                           | tablet     | font resizer,print |
+            | /news-events/cancer-currents-blog                                                           | mobile     | none,print         |
```

Comment:
I have a better approach for this - remove 'none' all together since you have a last scenario that verifies non-existence of font resizer

Reply:
Done